tests/functional/spec/thank_you.spec.js

```diff
@@ -53,10 +54,37 @@ describe("Thank You Default View Response Enabled", () => {
       $(DidYouKnowPage.submit()).click();
       $(ThankYouSubmitPage.submit()).click();
       expect(browser.getUrl()).to.contain(ThankYouPage.pageName);
+      expect($(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.false;
       expect($(ThankYouPage.title()).getHTML()).to.contain("Thank you for completing the Test Thank You");
       expect($(ThankYouPage.viewAnswersTitle()).getHTML()).to.contain("Get a copy of your answers");
       expect($(ThankYouPage.viewAnswersLink()).getText()).to.contain("save or print your answers");
       expect($(ThankYouPage.viewSubmittedWarning()).getHTML()).to.contain("For security your answers will only be available to view for 45 minutes");
     });
+
+    it("When I navigate to the thank you page, and I have submitted more than 45 minutes ago, Then I shouldn't see the option to view my answers", () => {
+      $(DidYouKnowPage.yes()).click();
+      $(DidYouKnowPage.submit()).click();
+      $(ThankYouSubmitPage.submit()).click();
+      expect(browser.getUrl()).to.contain(ThankYouPage.pageName);
+      browser.pause(46000); // Waiting 45 seconds for the timeout to expire (45 minute timeout changed to 45 seconds by overriding VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS for the purpose of the functional test)
+      browser.refresh();
+      expect($(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.true;
+      expect($(ThankYouPage.viewSubmittedGuidance()).getHTML()).to.contain("For security, you can no longer view your answers");
+    });
+
+    it("When I navigate to the view submitted response page, and click the save or print answers link, Then the 45 minute timeout passes, Then the information panel should replace the buttons on the page when the download button is clicked", () => {
```

Comment:
I think this should test should be in a new file, i.e `view_submitted_response.spec.js`, since it isn't about thank you.

I think the scenario needs some tidying. Maybe like:
- Given I am on the view submitted response page and I submitted over 45 minutes ago
- When I click the Download as PDF button
- Then I should be redirected to a page informing me that I can no longer view or get a copy of my answers.